Buttons and contact photo

dianadeayer opened the pull request:
Added save button, favorite icon and contact photo.
Added flex box to buttons and fixed the style.

![Screenshot 2025-04-15 at 17-02-38 Formulario](https://github.com/user-attachments/assets/1e043b7b-dc8b-42a4-a818-5efeabeaaca2)

dianadeayer committed:
avance css

dianadeayer committed:
second commit

josuemonroy2023 commented on index.html:
```diff
@@ -1,10 +1,115 @@
+          <div class="input-group phone-group">
+            <span class="material-icons">call</span>
+            <select class="country-code">
+              <option value="+52" selected>🇲🇽 +52</option>
```
We need to change the value of the options. We can use the short version of each country.

dianadeayer replied:
Values changed

josuemonroy2023 commented on index.html:
```diff
@@ -1,10 +1,115 @@
+        </div>
+
+        <div class="form-row">
+        <div class="input-group">
```
We need to fix the indentation.

dianadeayer replied:
Prettier was not working because of a syntax error, fiexed.

josuemonroy2023 commented on index.html:
```diff
@@ -1,10 +1,115 @@
+        <div class="input-group">
+          <span class="material-icons">cake</span>
+          <input type="text" id="day" placeholder="Day">
+       
```
We need to be sure the Prettier is used to fix the styles.

dianadeayer replied:
prettier fixed

josuemonroy2023 reviewed (changes requested):
There are some issues with the styles. Lets  make sure that we work on that.

josuemonroy2023 commented on style.css:
```diff
@@ -0,0 +1,173 @@
+  position: absolute;
+  top: 50%;
+  left: 10px;
+  transform: translateY(-50%);
```
Why are you using this transform with the translateY?

dianadeayer replied:
To move the label up the input container when focus is on

dianadeayer committed:
added buttons and photo icon

dianadeayer commented:
Added save button, favorite icon and contact photo.
Added flex box to buttons and fixed the style.

dianadeayer committed:
Label animation, css fix and photo add button

dianadeayer commented:
![image](https://github.com/user-attachments/assets/6dd7537c-e32c-4c03-ba40-c26ff131e69c)

Buttons now have the correct size of icons.
Added animation in labels, but the spaces between divs are now too close.
Indentation fixed.
Need to fix the font and padding of the labels.
Add button closer to photo card.

dianadeayer committed:
Fixed inputs, animations and allignments

dianadeayer commented:
> There are some issues with the styles. Lets make sure that we work on that.

styles fixed

dianadeayer commented:
![image](https://github.com/user-attachments/assets/fe5e7373-6a52-4b54-a202-1b882926e412)

Fixed animations, inputs alignments and paddings

dianadeayer commented:
![image](https://github.com/user-attachments/assets/7792c45e-c197-4726-b915-a25d4c0951ea)

1.-Fixed issues with body.
2.- Added arrow on header, but the header is not static yet. 
3.- Styles on select buttons are not fixed yet.
4.- Animations are completed, just need to fix the place holder of the birthday section.

dianadeayer committed:
Fixed body and styles

josuemonroy2023 commented on index.html:
```diff
@@ -1,10 +1,176 @@
+                class="contact-photo"
+                src="https://www.gstatic.com/identity/boq/profilepicturepicker/photo_silhouette_e02a5f5deb3ffc173119a01bc9575490.png"
+                alt="Contact Photo"
+                data-iml="8656"
```
What are we using this property for?

dianadeayer replied:
At first when I was looking for he source of the image in the goolge form code I thought it was part of it, as if it was something that "helped the image appear" but then I looked it up and from I understan its part of the google front end library and the number "8656" helps as an indicator of loading priority.

dianadeayer committed:
fixed add button and select styles

josuemonroy2023 reviewed (approved):
It looks good. There are a few style considerations to do, but we can review those later.

dianadeayer committed:
Cleaned CSS , fixed select styles and fixed photo add button

dianadeayer committed:
cleaned CSS and fixed header

dianadeayer commented:
![image](https://github.com/user-attachments/assets/1189c6f1-0492-489d-8a4e-584660e20e13)

I fixed the header, now when you scroll it keeps showing at the top of the page.
also changed my add buttons to flex end so they are more aligned with the input containers.

I checked my css to clean stuff that I wasnt using anymore from previews attempts from when I was trying to make the label animation work.

dianadeayer committed:
Fixed the 600px breakpoint , added 1200 and 900 and changed css order for better reading
